metricbeat/module/traefik/test_traefik.py

```diff
@@ -0,0 +1,28 @@
+import os
+import unittest
+import time
+from parameterized import parameterized
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../tests/system'))
```

Comment:
`import sys` :)

Reply:
Ahahaha, thank you.

Related: Is there some way to run system tests (i.e. the Python ones) for a specific module?

Reply:
`nosetests module/traefik/` should do the trick